### PR TITLE
Add New Featured Artist: Sobrem news post

### PR DESCRIPTION
**Pull request by Ephemeralis**
Damn.

### CODE REVIEW

**Comment by Ephemeralis** [comment]
Needs to wait on the upload of the MWC7K map before merge, for what it's worth.

**Review by peppy — DISMISSED** [review]

File: news/2023/2023-01-29-new-featured-artist-sobrem.md
Comment (LeoFLT) [inline news/2023/2023-01-29-new-featured-artist-sobrem.md:25]:
```suggestion
Check out [this map from the osu!mania 7K World Cup 2023](https://osu.ppy.sh/beatmapsets/1931456) hosted by [Jinjin](https://osu.ppy.sh/users/3360737) or [this avant-garde map](https://osu.ppy.sh/beatmapsets/955075) hosted by [rrtyui](https://osu.ppy.sh/users/352328)!
```

**Review by peppy — APPROVED** [review]

**Pull request merged**
